arangod/Graph/Enumerators/WeightedTwoSidedEnumerator.h

```diff
@@ -83,25 +165,25 @@ class WeightedTwoSidedEnumerator {
     [[nodiscard]] auto peekQueue() const -> Step const&;
     [[nodiscard]] auto isQueueEmpty() const -> bool;
     [[nodiscard]] auto doneWithDepth() const -> bool;
-    auto testDepthZero(Ball& other, CandidatesMap& results) -> void;
 
     auto buildPath(Step const& vertexInShell,
                    PathResult<ProviderType, Step>& path) -> void;
```

Comment:
How much work is it to change this from passing in a result that is filled to just returning the result?

Reply:
Uff. While sure this is possible - we always need to call that method twice to create a full path. 
This is the same way here now as we handle it in other type of enumerators. I do agree that we could do better - but I would not do it now in this PR. 

I'll create a ticket for this.

Reply:
https://arangodb.atlassian.net/browse/GORDO-1545